server-side/.gitignore

```diff
@@ -1,3 +1,3 @@
-node_modules
-.env
-package-lock.json
\ No newline at end of file
+node_modules/
+!uploads/.gitkeep
```

Comment:
.env
package-lock.json
should be in the gitIgnore file, don't remove them from here